### ISSUE TITLE
Is PIWAM discontinued ?

### ISSUE THREAD

**Issue by pousterlus** [piwam/piwam#9]
Hi,

I came accross this tool and seems to fit my needs.
But it seems discontinued... is it ?

-- 
Gorby.

**Comment by cyrilsuzat** [comment]
Yes, sorry. I did’t have time to maintain it. I plan to make a new version one day.

**Comment by AmauryVanEspen** [comment]
Hello  @cyrilsuzat 
Hope you're fine,
I've discovered you've set up a "new version" 3 years ago of Piwam, (from this post : https://groups.google.com/u/1/g/piwam)
great news.
With COVID, do you plan to publish a news version and fix the issue ?
Is there a demo somewhere with dummy dataset ?
Regards
Amaury

**Comment by cyrilsuzat** [comment]
Hello,

I haven't worked on this project for years.
I would nevertheless like to develop a new, more modern version as soon as I have the time.
Do not hesitate to contact me privately to give me your needs in order to guide and motivate the new project.
I will be delighted to know that associations are interested in such tools, which could motivate me to get back to it.

**Comment by AmauryVanEspen** [comment]
can we plan a call ? share or email or use mine please

Le lun. 16 nov. 2020 à 10:31, Cyril Suzat <notifications@github.com> a
écrit :

**Comment by cyrilsuzat** [comment]
@AmauryVanEspen  I contacted you through LinkedIn.

If other people are interested in this project, do not hesitate to contact me: https://cyrilsuzat.com
